templates/components/alert.html.twig

```diff
@@ -1,32 +1,29 @@
-{% macro alert(type, description, title, icon, dismissible) %}
+{% macro alert(options) %}
+    {% import '@Tabler/includes/utils.html.twig' as macros %}
 
-    {% if type is iterable %}
-        {% set options = type %}
+    {# Content #}
+    {% set _title = options.title ?? null %}
+    {% set _description = options.description ?? null %}
+    {% set _icon = options.icon ?? null %}
 
-        {% set _type = options.type ?? 'danger' %}
-        {% set _title = options.title ?? null %}
-        {% set _description = options.description ?? null %}
-        {% set _icon = options.icon ?? null %}
-        {% set _dismissible = (options.dismissible ?? false) is same as (true) %}
-        {% set _important = (options.important ?? false) is same as (true) %}
-        {% set _raw = (options.raw ?? false) is same as (true) %}
-        {% set _class = options.class is defined ? ' ' ~ options.class : '' %}
-    {% else %}
-        {% deprecated "Passing multiple parameters to 'alert()' Tabler macro is deprecated, use options object syntax." %}
+    {# Configuration #}
+    {% set _type = options.type ?? 'danger' %}
+    {% set _dismissible = (options.dismissible ?? false) is same as (true) %}
+    {% set _important = (options.important ?? false) is same as (true) %}
 
-        {% set _type = type ?? 'danger' %}
-        {% set _title = title is defined ? title|trans : null %}
-        {% set _description = description is defined ? description|trans : null %}
-        {% set _icon = icon ?? null %}
-        {% set _dismissible = (dismissible ?? false) is same as (true) %}
-        {% set _important = false %}
-        {% set _raw = false %}
-        {% set _class = '' %}
-    {% endif %}
+    {% set _id = options.id ?? null %}
+    {% set _attr = options.attr ?? {} %}
+    {% set _class = options.class ?? '' %}
+    {% set _raw = (options.raw ?? false) is same as (true) %}
 
     {% set _background = _title is empty or _description is empty or _important %}
 
-    <div class="alert alert-{{ _type }}{% if _dismissible %} alert-dismissible{% endif %}{% if _background %} alert-important{% endif %}{{ _class }}" role="alert">
+    <div
+            {%- if _id %}id="{{ _id }}"{% endif -%}
```

Comment:
That should do the trick
```suggestion
    <div {%- if _id %}id="{{ _id }}" {% endif -%}
```

Reply:
Produces that:
```
<divclass="alert alert-danger alert-dismissible alert-important "
        role="alert"
>
```

Reply:
Ah damn, I am stressed, sorry. I forgot to remove - 
```suggestion
    <div {% if _id %}id="{{ _id }}" {% endif -%}
```

Reply:
Wrapping all the code by `{% filter spaceless %}` doesn't change the attributes spaces of the `div` parent.

With:
```twig 
{{ alert({
    icon: 'help',
    title: 'title',
    description: 'description',
    dismissible: true,
    important: true,
}) }}
```

Produces:
```html
<div
                        class="alert alert-danger alert-dismissible alert-important "
            role="alert"
            
    ><div class="d-flex"><div class="alert-icon"><i class="icon far fa-question-circle"></i></div><div><h4 class="mb-1">title</h4><p class="m-0">description</p></div></div><a class="btn-close" data-bs-dismiss="alert" aria-label="close"></a></div>
```

Is it even possible to remove space inside HTML tag? 🤔

Reply:
Reverting that due to breaking code